model/src/pyrenew/process/rtperiodicdiff.py

```diff
@@ -203,11 +203,12 @@ def sample(
         )
 
         # Running the process
-        _, log_rt = lax.scan(
-            f=self.autoreg_process,
-            init=jnp.hstack([log_rt_prior, b]),
-            xs=noise,
-        )
+        ar_diff = FirstDifferenceARProcess(autoreg=b, noise_sd=noise)
```

Comment:
`noise_sd` should be `s_r`, not `noise`. `noise no longer needs to be sampled separately (it'll be sampled under the hood in `arr_diff`)

Reply:
Addressed in fe82dd1